mautic/mautic
# Add trello plugin

| Q                                      | A
| -------------------------------------- | ---
| Branch?                                | staging
| Bug fix?                               | no
| New feature?                           | yes
| Deprecations?                          | no
| BC breaks?                             | no
| Automated tests included?              | yes
| Related user documentation PR URL      | https://github.com/mautic/mautic-documentation/pull/99
| Related developer documentation PR URL | None
| Issue(s) addressed                     | None

#### Description:
Add basic functionality to create a Trello card from a Mautic contact. This is supposed to be an MVP, or a first basic release. It does the bare minimum of simply creating cards in Trello. 

Known limitations:
- no oAuth2.0
- no batch creation of cards

#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Set up authentification as described in the [documentation](https://github.com/mautic/mautic-documentation/pull/99)
3. Configure the favourite board as described in the [documentation](https://github.com/mautic/mautic-documentation/pull/99)
4. Create a new card from a contact detail page as described in the [documentation](https://github.com/mautic/mautic-documentation/pull/99)
5. Check if the card has been created in the corresponding list in Trello
